src/platform/PICO/io_def_generated.h

```diff
@@ -24,195 +24,193 @@
 // DEFIO_PORT_<port>_USED_COUNT is count of used pins on target
 
 #if defined(RP2350A)
-#define DEFIO_PORT_A_USED_COUNT 30
+#define DEFIO_USED_COUNT 30
```

Comment:
Maybe it is time to return to per-target available ping. It was mostly useless with unified targets, but with current hardware-specific targets, it can provide some error checking